src/app/components/action-card-component/action-card-component.component.html

```diff
@@ -0,0 +1,60 @@
+<div class="container">
+  <div class="card">
+    <div class="card-top">
+      <h5 class="card-title" [ngStyle]="{ color: titleColor }">
+        {{ title }}
+      </h5>
+      <button>
+        <i class="bi bi-plus-circle"></i>
+        {{ addActivityText }}
+      </button>
+    </div>
+
+    <div class="card-body">
+      <div *ngFor="let row of cardContentRow">
+        <ng-container>
```

Comment:
try adding *ngFor="let row of cardContentRow" to ng-container.
It would look sth like this: <ng-container *ngFor="let row of cardContentRow">
you will be able to reduce the code by 1 div